### PR TITLE
Updated minikube behavior

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

File: scripts/setup-minikube-for-linux.sh [hunk 1]
```diff
@@ -19,7 +19,7 @@ touch $HOME/.kube/config
 export KUBECONFIG=$HOME/.kube/config
 sudo -E minikube start --vm-driver=none
 sudo -E minikube addons enable ingress
-sudo -E chmod a+r ~/.minikube/client.key
+#sudo -E chmod a+r ~/.minikube/client.key
```
Comment (octonato) [inline scripts/setup-minikube-for-linux.sh:22]: why?

Comment (raboof) [reply scripts/setup-minikube-for-linux.sh:22]: apparently it's not longer there, and no longer needed...

**Review by octonato — APPROVED** [review]

**Comment by octonato** [comment]
@raboof, approved and merged, but maybe better to remove the line if no longer needed.

**Comment by raboof** [comment]
I kept it as a hint for people on an older version of minikube... though perhaps not worth it indeed. feel free to remove it.

**Comment by octonato** [comment]
Let's add comment then. I will send the PR, which version of minikube needs this?

**Comment by raboof** [comment]
I don't know

**Comment by octonato** [comment]
I see that this has been uncommentted on different places. I won't go chasing it now. Let's leave it like that then.

**Pull request merged**